### PR TITLE
feat(table row) - added info tooltip to landing page table row

### PR DESCRIPTION
**Pull request by sandymash**

### CODE REVIEW

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx [hunk 1]
```diff
@@ -156,6 +160,28 @@ const InvestigationTable: React.FC = (): JSX.Element => {
         return (<Popper {...props} style={{ width: 350 }} placement='bottom-start' />)
     }
 
+    const RowTooltip = (props: React.PropsWithChildren<{creationDate: Date; startTime: Date;}>) => {
+        const {creationDate, startTime} = props;
+        const noDataMessage = 'אין מידע אודות תאריכים לחקירה זו';
```
Comment (NaamanTsur) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:165]: You can place hard coded constants outside of the component.

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx [hunk 2]
```diff
@@ -156,6 +160,28 @@ const InvestigationTable: React.FC = (): JSX.Element => {
...
+        const formatDate = (date: Date) => format(new Date(date), 'dd/MM/yyyy');
+
+        const title = <>
```
Comment (NaamanTsur) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:168]: I think using `useMemo` here with creationDate and startTime is pretty good.

**Review by NaamanTsur — CHANGES_REQUESTED** [review]

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx (hunk 1 above)
Comment (sandymash) [reply client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx]: moved comp. outside and fixed

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx (hunk 2 above)
Comment (sandymash) [reply client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx]: sure

**Review by NaamanTsur — APPROVED** [review]

**Review by OrelKaravani — DISMISSED** [review]

**Review by eladyashar — APPROVED** [review]

**Review by OrelKaravani — APPROVED** [review]

**Review by NaamanTsur — APPROVED** [review]

**Pull request merged**